usage_report/annotations/annotations_fxhealth.json

```diff
@@ -77,6 +77,36 @@
         "pct_latest_version": "FF64"
       },
       "date": "2018-12-11"
+    },
+    {
+      "annotation": {
+        "MAU": "Summer Slump (Southern Hemisphere)"
```

Comment:
Nit: If we're putting `Southern Hemisphere` here, we should put it in the other ones for `Northern`. Generally I think we should just remove it.

Reply:
okay, I'll just call it summer slump.